### PR TITLE
Populate the initial scripts

### PR DESCRIPTION
**Pull request by lawsonamzn**
This is the initial population of the 3p package scripts from an internal repo to the public repo

### CODE REVIEW

**Review by lumberyard-employee-dm — APPROVED** [review]

**Review by spham-amzn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,269 @@
...
+        "python-3.7.5-darwin" : "python/darwin_x64/package"
+    }
+}
+```
+The "build_from_source" entry causes it to run ./python/build_package_image.py
+Note that you can specify parameters after the script name.
+
+The "build_from_folder" always executes after build from source, and will cause it to attempt to package up the contents of the ./python/darwin_x64/package folder (which was built using the script).
+
+## recommended package authoring workflow
```
Comment (amzn-changml) [inline README.md:253]: Nit: Capitalization

File: o3de_package_scripts/build_all_packages.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    FindPackageUtils.AddServerArgs(parser)
+    args = parser.parse_args()
+    CommonUtils.PostArgParse(args)
+
+    if args.profile_name:
+        print(f"Using AWS profile: {args.profile_name}")
+
+    if not args.server_urls:
+        print("You must specify server urls to check in order to use this script")
+        print("Either set LY_SERVER_URLS (semi colon list) or specify it in the command line in --server_urls (semi colon list)")
```
Comment (amzn-changml) [inline o3de_package_scripts/build_all_packages.py:144]: Should that be `LY_PACKAGE_SERVER_URLS` instead of `LY_SERVER_URLS`?

**Review by amzn-changml — APPROVED** [review]
I believe we addressed most of this before we moved to Github. Just some nitpicks found.

File: o3de_package_scripts/build_all_packages.py (hunk 2 above)
Comment (lawsonamzn) [reply o3de_package_scripts/build_all_packages.py:144]: Yeas!  I'll change it.  Thanks for spotting.

**Pull request merged**